pom.xml

```diff
@@ -829,6 +829,17 @@ under the License.</licenseText>
             </dependency>
           </dependencies>
         </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-pmd-plugin</artifactId>
```

Comment:
why do we have cost of carry? this plugin is not in use:
- https://github.com/apache/maven/pull/2331